packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -72,12 +72,10 @@ import {
   SubCategoryAppearance,
 } from "@itwin/core-common";
 import { Point3d, YawPitchRollAngles } from "@itwin/core-geometry";
-import {
-  IModelExporter,
-  IModelImporter,
-  IModelTransformer,
-  TransformerLoggerCategory,
-} from "../../transformer";
+import { IModelExporter } from "../../IModelExporter";
+import { IModelImporter } from "../../IModelImporter";
+import { IModelTransformer } from "../../IModelTransformer";
+import { TransformerLoggerCategory } from "../../TransformerLoggerCategory";
```

Comment:
tbh not sure I like this autofix lint rule... which one is it?
(I really hope it's not the barrel one I wrote :smile:)

Reply:
Seemed to not autofix it once I turned off the barrel one for the test dir.